recipe/meta.yaml

```diff
@@ -36,7 +36,8 @@ test:
 about:
   home: http://jupyter.org
   license: BSD 3-clause
-  summary: the base implementation of the Jupyter Notebook format
+  summary: Reference implementation of the Jupyter Notebook format
```

Comment:
Hmm...yeah, we have normally used `description` from `setup.py` for Python packages, but obviously that doesn't work for non-Python packages. So we recourse to other strategies. Probably we should document some best practices about metadata. In any event, thanks for updating this.